config/crd/bases/postgres-operator.crunchydata.com_pgadmins.yaml

```diff
@@ -973,24 +973,24 @@ spec:
                       More info: https://www.pgadmin.org/docs/pgadmin4/latest/external_database.html
                     properties:
                       key:
-                        description: The key of the secret to select from.  Must be
-                          a valid secret key.
+                        description: Name of the data field within the Secret.
+                        maxLength: 253
+                        minLength: 1
+                        pattern: ^[a-z0-9]([-a-z0-9]*[a-z0-9])?([.][a-z0-9]([-a-z0-9]*[a-z0-9])?)*$
```

Comment:
🔧 This isn't right! Subdomain length, yes, but the format isn't so limited.

https://github.com/kubernetes/apimachinery/blob/v0.27.0/pkg/util/validation/validation.go#L425-L441

Reply:
Amended.